src/electionguard_gui/web/components/admin/create-election-component.js

```diff
@@ -15,35 +15,29 @@ export default {
     Spinner,
   },
   methods: {
-    createElection() {
+    async createElection() {
```

Comment:
This was a somewhat unrelated refactoring that makes me happy.  When I wrote the create election component I wasn't as familiar with how to read files and call eel's methods with the async/await pattern.  I love this refactoring because it shows how much more readable things are with async/await.  It makes me very happy that Anders Hejlsberg introduced the pattern to the world in C# 5 in 2012 and it was so popular that JavaScript stole it five years later in 2017.